great_expectations/rule_based_profiler/util.py

```diff
@@ -242,36 +242,80 @@ def compute_quantiles(
         metric_values,
         q=(false_positive_rate / 2),
         axis=0,
-        interpolation="linear",  # can be omitted ("linear" is default)
```

Comment:
`interpolation` is deprecated in favor of `method`. This means using `interpolation` emits warnings and using `method` causes errors in Python 3.6 and 3.7. Since we are using the default method I have chosen to remove this parameter.